conform-pr/main.go

```diff
@@ -28,52 +28,70 @@ func main() {
 		action.Fatalf("main: %s", err)
 	}
 
-	errors := runChecks(action, client)
-
-	if len(errors) == 0 {
-		return
+	summaries := runChecks(action, client)
+	action.AddStepSummary("| Check  | Status |")
+	action.AddStepSummary("|--------|--------|")
+
+	for _, summary := range summaries {
+		statusSign := ":x:"
+		if summary.Ok {
+			statusSign = ":white_check_mark:"
+		}
+		if summary.Details != nil {
+			action.AddStepSummary(fmt.Sprintf("|%s | %s %s|", summary.Name, statusSign, summary.Details))
+		} else {
+			action.AddStepSummary(fmt.Sprintf("|%s | %s |", summary.Name, statusSign))
+		}
 	}
 
-	var serrors []string
-	for _, err := range errors {
-		serrors = append(serrors, fmt.Sprintf("%s", err))
+	for _, v := range summaries {
+		if !v.Ok {
+			action.Fatalf("The PR does not conform to the rules")
+		}
 	}
+}
 
-	action.Fatalf("The PR does not conform to the rules:\n - %s.", strings.Join(serrors, ";\n - "))
+// Summary is a markdown summary.
+type Summary struct {
+	Name    string
+	Ok      bool
+	Details error
```

Comment:
changed.

Reply:
No, it wasn't changed :)

The idea is that this fields would contain an error message; it could not contain some non-error detail message

Reply:
> No, it wasn't changed :)
> 
> The idea is that this fields would contain an error message; it could not contain some non-error detail message

renamed variable.